text/0001-cdk-update.md

```diff
@@ -0,0 +1,285 @@
+---
+rfc pr: [#335](https://github.com/aws/aws-cdk-rfcs/pull/335)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/1
+---
+
+# CDK Accelerate
+
+Shorten the development iteration speed -- the "edit-compile-test" loop -- for
+CDK applications.
+
+## Working Backwards
+
+### CHANGELOG
+
+- feat(cli): implement `cdk deploy --accelerated` and
+  `cdk deploy --accelerated --watch`
+
+### README
```

Comment:
Please add a section here that shows how the `cdk deploy --help` will describe this feature. Something like this:

```
cdk deploy --help

    --accelerated Perform accelerated deployment (`--accelerated=only` will fail if there are non-accelerated updates)
    -w --watch   Watch for file changes and deploy any updates. Implies --accelerated=auto


Examples:
    cdk deploy -w
    cdk deploy --accelerated
    cdk deploy --accelerated=only --watch
...    
```

Something like that.